src/libpriv/rpmostree-scripts.c

```diff
@@ -190,7 +190,7 @@ run_script_in_bwrap_container (int rootfs_fd,
                                /* Scripts can see a /var with compat links like alternatives */
                                "--ro-bind", "./var", "/var",
                                /* But no need to access persistent /tmp, so make it /tmp */
-                               "--bind", "/tmp", "/var/tmp",
+                               "--bind", "./tmp", "/var/tmp",
```

Comment:
Wait, I'm confused. Why do we even need to bind anything to `/var/tmp`? We already have `--dir /tmp` from the bwrap constructor, can't we get rid of this and the `mkdirat` above and just do `--dir /var/tmp` here too?